pagopa/pn-frontend
# Move client generation to the start, test and buil phases - pn-11372

## Short description
Remove auto generated clients from the repository, and generate them in the start, test and buil phases

## List of changes proposed in this pull request
- Removed auto generated directories
- Updated gitignore
- Updated dev scripts
- Updated sonar buildspec
- Updated cra buildspec

## How to test
- Run the start command and check that everything is ok
- Check the tests on aws pipeline
- Check the build process
